concrete-core/src/math/torus/mod.rs

```diff
@@ -81,15 +83,22 @@ pub trait UnsignedTorus:
     + FromTorus<f64>
     + IntoTorus<f64>
     + RandomGenerable<Gaussian<f64>>
+    + RandomGenerable<UniformBinary>
+    + RandomGenerable<UniformTernary>
     + RandomGenerable<Uniform>
     + Display
     + Debug
-    + CastFrom<bool>
     + CastFrom<f64>
     + CastInto<f64>
 {
+    /// The log standard deviation used to sample gaussian keys in this precision.
+    const GAUSSIAN_KEY_LOG_STD: LogStandardDev;
```

Comment:
In order for the gaussian sampling code to be generic over the different precisions, we store here the log standard deviation const.